### PR TITLE
fix: sort containers/images by CreatedAt

### PR DESCRIPTION
**Pull request by xyz-li**
Sort containers by CreatedAt.

### CODE REVIEW

**Comment by xyz-li** [comment]
@davidhsingyuchen 
@apostasie

**Comment by xyz-li** [comment]
This is the default behavior of docker.

**Comment by apostasie** [comment]
Thanks @xyz-li  - looks good to me!

Do you think you could add a few tests for these?

**Comment by AkihiroSuda** [comment]
Please squash the commits

File: pkg/cmd/image/list.go [hunk 1]
```diff
@@ -108,9 +109,17 @@ func List(ctx context.Context, client *containerd.Client, filters, nameAndRefFil
 
 		imageList = imgutil.FilterImages(imageList, beforeImages, sinceImages)
 	}
+
+	sort.Sort(byCreatedAt(imageList))
```
Comment (AkihiroSuda) [inline pkg/cmd/image/list.go:113]: Can't we use `sort.Slice` here?

**Comment by AkihiroSuda** [comment]
@xyz-li Do you plan to add tests?

I guess adding tests is optional (still recommended though) for this kind of change.

**Comment by xyz-li** [comment]
> @xyz-li Do you plan to add tests?
> 
> I guess adding tests is optional (still recommended though) for this kind of change.

I will do it next week.

**Comment by xyz-li** [comment]
> @xyz-li Do you plan to add tests?
> 
> I guess adding tests is optional (still recommended though) for this kind of change.

Done.

**Review by AkihiroSuda — APPROVED** [review]
Thanks

**Pull request merged**